osfoffline/settings/defaults.py

```diff
@@ -47,3 +47,9 @@
 
 PROJECT_LOG_DIR = user_log_dir(appname=PROJECT_NAME, appauthor=PROJECT_AUTHOR)
 PROJECT_LOG_FILE = os.path.join(PROJECT_LOG_DIR, 'osfoffline.log')
+
+#updater
+REPO = 'CenterForOpenScience/OSF-Offline'
+VERSION = '0.2.1'
+NAME = 'OSF-Offline'
+MIN_VERSION_URL = 'https://raw.githubusercontent.com/CenterForOpenScience/OSF-Offline/develop/deploy/Offline-version.json'
```

Comment:
This is adjusted from https://raw.githubusercontent.com/chennan47/OSF-Offline/feature/auto_updater/deploy/Offline-version.json